### ISSUE TITLE
Latest Date Metadata Field

### ISSUE THREAD

**Issue by SrinivasRao17** [DSpace/dspace-angular#2268]
Good Morning All,

Can you please help me with Metadata Field which identify the current changed date in the UI. 

For example if there is any changes in the description or title modification then date has to be modified or updated. 

We tried with dc.date.updated in UI but there is no changes. Please suggest. Thanks!!

**Comment by SrinivasRao17** [comment]
@tdonohue Can you please help me with the above request. Thanks!!

**Comment by tdonohue** [comment]
I don't believe any such metadata field exists in DSpace 7 at this time.  This is not a feature that has ever existed in DSpace.  It would require custom coding to implement such a feature for all object types.

This ticket doesn't appear to be actionable, and it's more of a question.  Keep in mind questions should be asked using our Support options: https://wiki.lyrasis.org/display/DSPACE/Support  Tickets should only be created for logging bugs or creating new feature requests.

Closing this as the question is answered.
